test/e2e/fleet_test.go

```diff
@@ -223,14 +223,16 @@ func TestFleetRollingUpdate(t *testing.T) {
 	t.Parallel()
 	ctx := context.Background()
 	// Use scaleFleetPatch (true) or scaleFleetSubresource (false)
-	fixtures := []bool{true, false}
-	maxSurge := []string{"25%", "10%"}
+	fixtures := []bool{true}    // , false} // TODO Enable these again
+	maxSurge := []string{"25%"} // , "10%"} // TODO
+	doCycle := true             // TODO: fixture?
```

Comment:
`doCycle` should be a fixture as well, but that would double the number of tests to run from 4 to 8? Doesn't seem great, especially since they're all running in parallel?